pkgs/development/tools/poetry2nix/poetry2nix/README.md

```diff
@@ -0,0 +1,2 @@
+Dont change these files here, they are maintained at:
```

Comment:
While development is done there, do describe the process for updating these expressions
pkgs/development/tools/poetry2nix/update